### ISSUE TITLE
helper get_credentials_from_store (client.py) does not seem used

### ISSUE THREAD

**Issue by zogzog** [dulwich/dulwich#798]
It's a very new and welcome addition but it seems to be sitting unused.
An oversight of some sort ?

Thanks in advance,
Aurélien

**Comment by zogzog** [comment]
Maybe Manuel put it there for later use only from hg-git ?

**Comment by jelmer** [comment]
Yeah, at the moment it's just a convenience function that applications can call out to. At some point we may want to hook it into the porcelain.
